### PR TITLE
Changed colors to support Dark Mode

### LINKED ISSUE #23: Dark mode support

### PR DESCRIPTION
**Pull request by abanobmikaeel**
Fixes #23

### CODE REVIEW

**Comment by schayes04** [comment]
The yellow with white text is not readable

Screen Shot 2019-12-17 at 2 56 06 PM

**Comment by schayes04** [comment]
Searching on the full detail page has all black text

Screen Shot 2019-12-17 at 2 56 15 PM

**Comment by schayes04** [comment]
Update from master to resolve the CI failing

**Comment by bfwalton** [comment]
image
Looks like this header is black when in light mode, should probably match other colors and be white

File: Sources/LogView/LogTableViewController.swift [hunk 1]
```diff
@@ -34,6 +34,18 @@ class LogTableViewController: UIViewController {
     var selectedLogItem: LogItem!
     var domainSections: [CollapsibleSection<LogItem>] = []
     
+    override var preferredStatusBarStyle: UIStatusBarStyle {
```
Comment (schayes04) [inline Sources/LogView/LogTableViewController.swift]: Bad merge conflict? I think all these added lines should have been removed

Comment (abanobmikaeel) [reply Sources/LogView/LogTableViewController.swift]: I believe so yeah, will remove!

**Pull request merged**